kimchi/wasm/src/plonk_verifier_index.rs

```diff
@@ -20,6 +20,68 @@ use std::path::Path;
 use std::sync::Arc;
 use wasm_bindgen::prelude::*;
 
+#[wasm_bindgen]
+pub struct WasmLookupPatterns {
+    inner: LookupPatterns,
+}
+
+#[wasm_bindgen]
+impl WasmLookupPatterns {
+    #[wasm_bindgen(constructor)]
+    pub fn new(xor: bool, lookup: bool, range_check: bool, foreign_field_mul: bool) -> Self {
+        WasmLookupPatterns {
+            inner: LookupPatterns {
+                xor,
+                lookup,
+                range_check,
+                foreign_field_mul,
+            },
+        }
+    }
+}
+
+#[wasm_bindgen]
+pub struct WasmLookupFeatures {
+    inner: LookupFeatures,
+}
+
+#[wasm_bindgen]
+impl WasmLookupFeatures {
+    #[wasm_bindgen(constructor)]
+    pub fn new(
+        patterns: &WasmLookupPatterns,
+        joint_lookup_used: bool,
+        uses_runtime_tables: bool,
+    ) -> Self {
+        WasmLookupFeatures {
+            inner: LookupFeatures {
+                patterns: patterns.inner.clone(),
+                joint_lookup_used,
+                uses_runtime_tables,
+            },
+        }
+    }
+}
+
+#[wasm_bindgen]
+pub struct WasmLookupInfo {
+    inner: LookupInfo,
+}
+
+#[wasm_bindgen]
+impl WasmLookupInfo {
+    #[wasm_bindgen(constructor)]
+    pub fn new(max_per_row: usize, max_joint_size: u32, features: &WasmLookupFeatures) -> Self {
+        WasmLookupInfo {
+            inner: LookupInfo {
+                max_per_row,
+                max_joint_size,
+                features: features.inner.clone(),
+            },
+        }
+    }
+}
```

Comment:
You should not have to do that for The WASM bindings for all of them are generated in `kimchi` (https://github.com/o1-labs/proof-systems/pull/1182 for develop, https://github.com/o1-labs/proof-systems/pull/1186 for berkeley)